examples/fscloud/variables.tf

```diff
@@ -57,3 +57,9 @@ variable "allow_vpcs_to_cos" {
   description = "Set rule for VPCs to COS, deafult is true"
   default     = true
 }
+
+variable "location" {
```

Comment:
Put the default inline (with some comments around them) and remove the variable

Reply:
done